value/src/main/java/com/google/auto/value/processor/AutoValueProcessor.java

```diff
@@ -410,26 +427,24 @@ private void processType(TypeElement type) {
     }
   }
 
-  private AutoValueExtension.Context makeExtensionContext(
-      final TypeElement type, final Iterable<ExecutableElement> methods) {
-    return new AutoValueExtension.Context() {
-
-      @Override public ProcessingEnvironment processingEnvironment() {
-        return processingEnv;
-      }
-
-      @Override public String packageName() {
-        return TypeSimplifier.packageNameOf(type);
-      }
-
-      @Override public TypeElement autoValueClass() {
-        return type;
-      }
+  private <T> ImmutableSet<T> newFilteredImmutableSet(ImmutableSet<T> original,
+                                                      Predicate<T> predicate) {
```

Comment:
the formatting is off here, not matching the rest of the project (disable "Align when multiline" for "Method call arguments" in your editor code style).